src/Synth/OscilGen.cpp

```diff
@@ -473,6 +474,25 @@ void OscilGen::convert2sine()
     prepare();
 }
 
+class timer_t
+{
+    using time_point = std::chrono::steady_clock::time_point;
+    std::size_t num;
+    time_point begin;
+public:
+    timer_t(std::size_t num) :
+        num(num),
+        begin(std::chrono::steady_clock::now()) {}
```

Comment:
Some of this code will likely have to be reworked at some stage as there's mingw issues with std::chrono. For the time being though I wouldn't worry about it too much. I forget, do I have a build setup for mingw on the main repo yet?

If not I have a docker file over with the zyn-fusion-build scripts which I know was testing the pipeline.

Reply:
I see only 3x ubuntu. I'll leave the comment open for later.

Reply:
Ok, I'll have to port over some of those bits. It's a much slower process given that dependencies need to be compiled, so non-ideal for full continuous integration. Seems like one of those things which would be better with a custom base docker image with the deps already built, but I'm still very much a docker novice. 

A quick googling makes it look possible, so it might be one of those things to setup the sloppy approach first and then pivot to having a more optimized setup a bit later.